lib/sycamore/sycamore/llms/prompts/default_prompts.py

```diff
@@ -463,6 +474,154 @@ def __init__(self, question: str, text: str):
         )
 
 
+SummarizeBranchingFactorJinjaPrompt = JinjaElementPrompt(
+    system=textwrap.dedent(
+        """
+        {% if question is defined %}You are a helpful research assistant. You answer questions based on text you are presented with.
+        {% else %}You are a helpful data summarizer. You concisely summarize text you are presented with, including as much detail as possible.
+        {% endif %}
+        """
+    ),
+    user=J_GET_ELEMENT_TEXT_MACRO
```

Comment:
Can we force the question to always be present by having a default question of "What is the summary of this information?"

This feels overly complex, and experience with FINRA was that complexity leads to weird prompts that don't quite do what you want.